tests/tests.py

```diff
@@ -375,6 +376,33 @@ def test_expired_datetime_offset(self):
         entry.expiry_time = past_time
         self.assertTrue(entry.expired)
 
+    def test_touch(self):
+        """Test for https://github.com/pschmitt/pykeepass/issues/120"""
+        entry = Entry(
+            'title',
+            'username',
+            'password',
+            kp=self.kp
+        )
+        atime = entry.atime
+        mtime = entry.mtime
+        ctime = entry.ctime
+        time.sleep(1)
```

Comment:
I think `atime`, `ctime`, and `mtime` all record microseconds, so can't we get away with a much smaller sleep?